easybuild/easyconfigs/t/testpath/testpath-0.3-goolf-1.7.20-Python-2.7.11.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'testpath'
+version = '0.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/jupyter/testpath'
+description = """Test utilities for code working with files and commands"""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['https://pypi.python.org/packages/py2.py3/t/testpath/']
+sources = ['testpath-%(version)s-py2.py3-none-any.whl']
+
+dependencies = [
+    ('Python', '2.7.11'),
+]
+
+#builddependencies = [
+#    ('pip', '8.1.2', versionsuffix),
+#]
```

Comment:
ah, wait, we have `pip` as a part of the `Python` already, right? so just drop this